mmv1/products/compute/api.yaml

```diff
@@ -3285,15 +3285,15 @@ objects:
           are 4096 and 16384, other sizes may be added in the future.
           If an unsupported value is requested, the error message will list
           the supported values for the caller's project.
-      - !ruby/object:Api::Type::Enum
+      - !ruby/object:Api::Type::String
         name: 'interface'
         min_version: 'beta'
+        # interface is removed using url_param_only to preserve schema definition
+        # and prevent sending or reading in API requests
+        url_param_only: true
+        deprecation_message: This field is no longer in use, disk interfaces will be automatically determined on attachment.
         description: |
           Specifies the disk interface to use for attaching this disk, which is either SCSI or NVME. The default is SCSI.
-        default_value: :SCSI
```

Comment:
This is unsafe to remove, right? It'll cause a diff from `"SCSI"` -> `""` for anyone using it through the default value.

Reply:
Oh, yeah that would be a disaster. Good catch!